Sources/FluentPostgreSQL/PostgreSQLDriver.swift

```diff
@@ -2,7 +2,9 @@ import Fluent
 import PostgreSQL
 
 public class PostgreSQLDriver: Fluent.Driver {
-    public var idKey: String = "id"
+    
+    internal static let idKey: String = "id"
+    public var idKey: String = PostgreSQLDriver.idKey
```

Comment:
I think there's an issue here having two unbound values intended to represent one thing (unless I'm mistaken). If `idKey` is a let at static level, and not at instance level, then we run the risk of the two diverging and implementations will very likely get weird results.

Making static version settable w/ this as instance variation seems better, let me know if there's something I'm not seeing.

```swift
var idKey: String { return PostgreSQLDriver.idKey }
```